build.psm1

```diff
@@ -446,6 +446,32 @@ Fix steps:
         Pop-Location
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-linux`
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @(@{InputObject=New-PSBreakpoint; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 540
540:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Warning; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 564
564:         $result | Should -BeNullOrEmpty
```
Experimental Feature Basic Tests - Feature-Disabled.No experimental feature is enabled
```powershell
Expected 0, but got 3.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 40
40:         $EnabledExperimentalFeatures.Count | Should -Be 0
```